ppr-ui/src/components/registration/securities-act-notices/AddEditCommissionOrder.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <FormCard
+    id="add-edit-commission-order"
```

Comment:
Not a big deal (since we used it across many components already), but I ran into scenario when same component is used multiple times on the page, and they all have same ids. Quick workaround was to overwrite the id (when adding component to template), but maybe we can think of some long term solution.

Reply:
I see, and what were the implications?
Each component should have an identifier at it's root level, imo. It's a practice we've always followed.

I think what you did is a viable solution for multiple instances of a single component, very much the same in this example. 
FormCard for example is a common component, in this case, we've applied an identifier unique to this implementation. 

I'm not sure there is a catch all solution but definitely open to your thoughts!